test/caching-precompiler.js

```diff
@@ -145,6 +145,6 @@ test('uses babelConfig for babel options when babelConfig is an object', functio
 	t.true('inputSourceMap' in options);
 	t.false(options.babelrc);
 	t.same(options.presets, ['stage-2', 'es2015']);
-	t.same(options.plugins, [customPlugin, powerAssert, transformRuntime, rewrite]);
+	t.same(options.plugins, [customPlugin, transformRuntime, rewrite, powerAssert]);
```

Comment:
Ideally, powerAssert would run earlier so it captures source as written, instead of as transformed.

https://gist.github.com/jamestalmage/885e1d0d732557cfb5cf854aee04fd19?ts=2